bcda/api/v2/api.go

```diff
@@ -88,3 +95,114 @@ func BulkGroupRequest(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+
+/*
+	swagger:route GET /api/v2/metadata metadata metadata
+
+	Get metadata
+
+	Returns metadata about the API.
+
+	Produces:
+	- application/fhir+json
+
+	Schemes: http, https
+
+	Responses:
+		200: MetadataResponse
+*/
+func Metadata(w http.ResponseWriter, r *http.Request) {
```

Comment:
Does the new endpoint need to be registered in the router?  For example: https://github.com/CMSgov/bcda-app/blob/master/bcda/web/router.go#L40-L45